### PR TITLE
Rework ReadMCURegistersAsync such that it does not ping the dead conn…

### PR DESCRIPTION
**Pull request by Liam920**
…ection to the MCU (causes mscorlib errors). Then modify the routine to ping every 2 seconds, then every 10 after the first 10.

### CODE REVIEW

**Review by robbycw — APPROVED** [review]
This all looks good to me. 2 seconds / 10 seconds seem like appropriate wait times before attempting to reconnect. It's a shame we can't get rid of the exception message from the NModbus library's read method.

File: ControlRoomApplication/ControlRoomApplication/Controllers/PLCCommunication/PLCDrivers/MCUManager/MCUManager.cs [hunk 1]
```diff
@@ -34,6 +34,7 @@ public class MCUManager {
         private TcpClient MCUTCPClient;
         private MCUConfigurationAxys Current_AZConfiguration;
         private MCUConfigurationAxys Current_ELConfiguration;
+        private bool MCUDisconnected = false;
```
Comment (jsnyder17) [inline ControlRoomApplication/ControlRoomApplication/Controllers/PLCCommunication/PLCDrivers/MCUManager/MCUManager.cs:37]: trash code. lgtm

**Review by Liam920 — COMMENTED** [review]
lgtm

**Review by jsnyder17 — APPROVED** [review]
trash code. lgtm

**Pull request merged**